model/task/db.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/evergreen-ci/evergreen/util"
 	"github.com/mongodb/anser/bsonutil"
 	"github.com/pkg/errors"
-	"gopkg.in/mgo.v2"
+	mgo "gopkg.in/mgo.v2"
```

Comment:
I don't think this is necessary, but honestly have never thought about this before. Was this done by your editor or by you?

Reply:
the editor.

Reply:
I think goimport does this because having the package name verbatim would not compile

Reply:
This did compile though, and my goimport hook doesn't do this. I think it's fine to merge either way though.